borsar/cluster/obj.py

```diff
@@ -33,7 +33,12 @@ def read_cluster(fname, subjects_dir=None, src=None, info=None):
     clst : Clusters
         Cluster results read from file.
     '''
-    from mne.externals import h5io
+    try:
+        # mne < 1.0
+        from mne.externals import h5io
+    except ModuleNotFoundError:
+        # mne > 1.0 requires separate installation of h5io
+        import h5io
```

Comment:
this repeats a few times - we could wrap this conditional import into a function that returns the correct module.

Reply:
True :) can you do that in a separate PR though?

Reply:
sure!